### PR TITLE
kni: nrop: kickoff branch 4.16

### PR DESCRIPTION
**Pull request by ffromani**
create release-4.15 branch from main branch,
redirect main branch to 4.16.

### CODE REVIEW

**Comment by ffromani** [comment]
the mirror failure look legit, I'll check

**Comment by ffromani** [comment]
/pj-reharse ack

failure not critical

File: core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # numaresources-operator
-registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator quay.io/openshift-kni/numaresources-operator:4.15.999-snapshot
-registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator-tests quay.io/openshift-kni/numaresources-operator-tests:4.15.999-snapshot
+registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator-tests quay.io/openshift-kni/numaresources-operator-tests:4.16.999-snapshot
```
Comment (yanirq) [inline core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay:2]: numaresources-operator-tests is always just for the latest branch ?

Comment (ffromani) [reply core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay:2]: yes

**Pull request merged**